x-pack/legacy/plugins/siem/cypress/support/commands.js

```diff
@@ -1,31 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-// ***********************************************
-// This example commands.js shows you how to
-// create various custom commands and overwrite
-// existing commands.
-//
-// For more comprehensive examples of custom
-// commands please read more here:
-// https://on.cypress.io/custom-commands
-// ***********************************************
-//
-//
-// -- This is a parent command --
-// Cypress.Commands.add("login", (email, password) => { ... })
-//
-//
-// -- This is a child command --
-// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
-//
-//
-// -- This is a dual command --
-// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
-//
-//
-// -- This is will overwrite an existing command --
-// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
```

Comment:
Just a tip (no changes requested)... When you encounter files that you need to convert from `js` to `ts` or just need to rename them in general `git` has a `git mv` command which will preserve the history of the file and cause no conflicts for other devs if they happen to be working on a file which was renamed.

Without doing a `git mv`, what can happen is another dev could be working within this file and suddenly find it looks to be deleted and get a nasty git conflict with a deleted file when they merge from master.

I think you're safe here as I don't think others are working in this file, just pointing it out.